api/app/settings/common.py

```
@@ -928,6 +928,7 @@
     "PASSWORD_RESET_CONFIRM_URL": "password-reset/confirm/{uid}/{token}",
     # if True user required to click activation link in email to activate account
     "SEND_ACTIVATION_EMAIL": env.bool("ENABLE_EMAIL_ACTIVATION", default=False),
+    "EMAIL_FRONTEND_DOMAIN": DOMAIN_OVERRIDE,
```

Ruff (0.16.1)

[error] Undefined name `DOMAIN_OVERRIDE`

(F821)